azext_edge/edge/_help.py

```diff
@@ -1219,3 +1219,81 @@ def load_iotops_help():
+        long-summary: |
+          The trusted certificate list contains the certificates of all the OPC UA servers that the connector
+          for OPC UA trusts. If the connector for OPC UA trusts a certificate authority, it automatically trusts
+          any server that has a valid application instance certificate signed by the certificate authority.
+          See the following resource for more info https://aka.ms/overview-opcua-broker-certificates-management
+    """
+
+    helps[
+        "iot ops connector opcua trust add"
+    ] = """
+        type: command
+        short-summary: Add a trusted certificate to the OPC UA Broker's trusted certificate list.
+        long-summary: |
+            The certificate file extension must be .der or .crt.
+        examples:
+        - name: Add a trusted certificate to the OPC UA Broker's trusted certificate list.
+          text: >
+            az iot ops connector opcua trust add --instance instance --ig instanceresourcegroup --cf certificate_file
+        - name: Add a trusted certificate to the OPC UA Broker's trusted certificate list with custom secret name.
+          text: >
+            az iot ops connector opcua trust add --instance instance --ig instanceresourcegroup --cf certificate_file
+            -s custom_secret_name
+    """
+
+    helps[
+        "iot ops connector opcua issuer"
+    ] = """
+        type: group
+        short-summary: Manage issuer certificates for the OPC UA Broker.
+        long-summary: |
+          The issuer certificate list stores the certificate authority certificates that the connector for OPC UA trusts.
+          This list can be used to manage the trust relationship if OPC UA server's application instance certificate is
+          signed by an intermediate certificate authority, but don't want to automatically trust all the certificates
+          issued by the certificate authority.
+          See the following resource for more info https://aka.ms/overview-opcua-broker-certificates-management
+    """
+
+    helps[
+        "iot ops connector opcua issuer add"
+    ] = """
+        type: command
+        short-summary: Add an issuer certificate to the OPC UA Broker's issuer certificate list.
+        long-summary: |
+            The certificate file extension must be .der, .crt or .crl. When adding a .crl file, a .der or .crt file with
+            same file name must be added first.
+        examples:
+        - name: Add an issuer certificates in the OPC UA Broker's trusted certificate list.
+          text: >
+            az iot ops connector opcua issuer add --instance instance --ig instanceresourcegroup --cf certificate_file
+        - name: Add an issuer certificate to the OPC UA Broker's issuer certificate list with custom secret name.
+          text: >
+            az iot ops connector opcua issuer add --instance instance --ig instanceresourcegroup --cf certificate_file
```

Comment:
btw if there is no other resource resource group used - you can just replace instance-resource-group with resource-group

Reply:
if we change to just resource-group, I'm a bit worried about the user would not know this is for instance's rg and they just put whatever rg they want, then we will have an issue since we cannot get instance correctly and people don't read help😤